patches/third_party-WebKit-Source-modules-donottrack-NavigatorDoNotTrack.cpp.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/third_party/WebKit/Source/modules/donottrack/NavigatorDoNotTrack.cpp b/third_party/WebKit/Source/modules/donottrack/NavigatorDoNotTrack.cpp
+index 74a03550c214818a8e9fd58bc12bd33fb62b8959..57cf0b29311bb41aff6e05854c26bbe08af67299 100644
+--- a/third_party/WebKit/Source/modules/donottrack/NavigatorDoNotTrack.cpp
++++ b/third_party/WebKit/Source/modules/donottrack/NavigatorDoNotTrack.cpp
+@@ -56,7 +56,11 @@ NavigatorDoNotTrack& NavigatorDoNotTrack::From(Navigator& navigator) {
+ }
+ 
+ String NavigatorDoNotTrack::doNotTrack(Navigator& navigator) {
++#if defined(MUON_CHROMIUM_BUILD)
+   return NavigatorDoNotTrack::From(navigator).doNotTrack();
```

Comment:
do not track can be managed from a preference

Reply:
prefs::kEnableDoNotTrack

Reply:
The intent here is to let the feature to work, but not return the status if `DoNotTrack` is enabled or not using the navigator api in js. cc: @diracdeltas

Reply:
i'm also confused what this is doing. the code in browser laptop navigator.js is to make DNT work correctly in js, not to disable it.